docs/docs/howto/clusterconnectivity/aiven-kafka-connect-cluster-ssl-protocol.rst

```diff
@@ -29,28 +29,12 @@ Follow the steps below to configure and connect an Aiven for Apache Kafka Connec
 4. Click **Save**. 
```

Comment:
This is not from this commit but from -prerequistes above until "two APIs can communicate over https." Its not formatted correctly.

Reply:
done.